src/request/sign-transaction/SignTransaction.js

```diff
@@ -1,8 +1,69 @@
 /* global Nimiq */
 /* global TransactionType */
 /* global KeyStore */
-/* global Utf8Tools */
+/* global Identicon */
+
 class SignTransaction { // eslint-disable-line no-unused-vars
+    /**
+     * @param {TransactionRequest} txRequest
+     * @returns {HTMLDivElement}
+     */
+    fillTransactionDetails(txRequest) {
+        /** @type {HTMLDivElement} */
+        const $transaction = (document.querySelector('#transaction-data .transaction'));
```

Comment:
are the parentheses needed?

Reply:
Yes, for type-casting for typescript, as otherwise the `querySelector` result can be `null`.